### ISSUE TITLE
Progress Bars?

### ISSUE THREAD

**Issue by ericbarefoot** [DeltaRCM/DeltaMetrics#5]
Some of the operations that we may end up doing on these data cubes could be quite time-consuming. Or, alternatively, the user could f* up and not know that the thing they requested was dumb.

Progress bars can be helpful for stuff like this, and I like them, in particular [this package called `tqdm`](https://tqdm.github.io/). What's the vibe on incorporating something like `tqdm` into our package here, and allowing the user to turn them on or off, or maybe specify a verbosity level, like we do in pyDeltaRCM.

**Comment by amoodie** [comment]
Yeah, I'd be fine with a command line progress bar, no GUI junk though. 

Maybe for now there is some way we can half-implement this as we write, so it becomes easier to implement down the line? Like always declaring `nt` and `t` in top-level loops or something?

That said, I haven't needed longer than a half second or so to compute the stratigraphy on any of the pyDeltaRCM cubes I've made (`3000 x 6000 x 5000` cells). Maybe we can worry about this when it becomes a problem.
